src/libraries/System.IO.Ports/Directory.Build.props

```diff
@@ -4,5 +4,6 @@
     <StrongNameKeyId>Open</StrongNameKeyId>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
     <UnsupportedOSPlatforms>browser</UnsupportedOSPlatforms>
+    <PackageVersion>5.0.1</PackageVersion>
```

Comment:
Given no change to the assembly I am OK without AssemblyVersion bump here, but normally we'd bump it.

Reply:
> Given no change to the assembly I am OK without AssemblyVersion bump here, but normally we'd bump i

Thats why kept it the same.